### ISSUE TITLE
Provide RPMs for ucode

### ISSUE THREAD

**Issue by Akira25** [jow-/ucode#243]
I invested some effort into providing an rpm-package for ucode. I'm a bit undecisive, in what could happen with that next, but wanted to let you know, that I managed to get an spec-file for fedora building on several architectures.

Next steps might be, to automatise the build somehow, which could be achieved in several ways. Maybe going upstream into fedora package feeds might be an option.

https://copr.fedorainfracloud.org/coprs/mhuebner/ucode/

Have fun and thank you for that handy language!

**Comment by ynezz** [comment]
> I invested some effort into providing an rpm-package for ucode

Awesome, it would be really great to `dnf install ucode` on Fedora based systems some day in the future.

FYI there is already a packaging for Debian in the [debian](https://github.com/jow-/ucode/blob/master/debian) folder and accompanying [.github/workflows/debian.yml](https://github.com/jow-/ucode/blob/master/.github/workflows/debian.yml) workflow for CI/QA, so maybe something to follow?

**Comment by Akira25** [comment]
> > I invested some effort into providing an rpm-package for ucode
> 
> Awesome, it would be really great to `dnf install ucode` on Fedora based systems some day in the future.

Yep, I'd really like that too, but experienced the process of contributing software packages to fedora as being far away from easy and straight forward. :/

For now, it would be enough, to spawn three commands:
```
sudo dnf install dnf-plugins-core  # (if not installed already)
sudo dnf copr enable mhuebner/ucode
sudo dnf install ucode
```

> 
> FYI there is already a packaging for Debian in the [debian](https://github.com/jow-/ucode/blob/master/debian) folder and accompanying [.github/workflows/debian.yml](https://github.com/jow-/ucode/blob/master/.github/workflows/debian.yml) workflow for CI/QA, so maybe something to follow?

Thanks for the hint. I tried to build something similar for rpm-packages, but unfortunately, there is no fedora docker container for the github ci. Instead, one might automate the package building via a webhook, which is supported by copr.

It would start building for all the selected architectures and fedora versions, once there is a new release (but this isn't realised in my repo yet).
